src/Cdn/Cdn/Models/Endpoint/PSUrlFileExtensionConditionParameters.cs

```diff
@@ -1,26 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
```

Comment:
@hytao was this type being used anywhere?

Reply:
It was used in ModleExtensions.cs. This type has been removed from latest swagger and .net SDK

Reply:
This was not an outputType or parameter for any cmdlet. I think it should be fine to remove it.